main.go

```diff
@@ -66,6 +66,11 @@ func bindRootFlags(rootCmd *cobra.Command) error {
 	if err != nil {
 		return err
 	}
+	rootCmd.Flags().StringSliceP("required-scopes", "r", nil, "force scope to be one of")
```

Comment:
I wouldn't use the shorthand here. https://godoc.org/github.com/spf13/pflag#StringSlice could be used since `r` is not that useful.

Reply:
adjusted